doc/howto/cluster_config_networks.md

```diff
@@ -0,0 +1,103 @@
+       lxc network create --target node3 my-network
+
+   ```{note}
+   You can pass only the member-specific configuration keys `bridge.external_interfaces`, `parent`, `bgp.ipv4.nexthop` and `bgp.ipv6.nexthop`.
+   Passing other configuration keys results in an error.
+   ```
+
+   These commands define the network, but they don't create it.
+   If you run `lxc network list`, you can see that the network is marked as "pending".
+1. Run the following command to instantiate the network on all cluster members:
+
+       lxc network create my-network
+
+   ```{note}
+   You can add configuration keys that are not member-specific to this command.
+   ```
+
+   If you missed a cluster member when defining the network, or if a cluster member is down, you get an error.
+
+Also see {ref}`network-create-cluster`.
+
+## Separate REST API and clustering networks
+
+You can configure different networks for the REST API endpoint of your clients and for internal traffic between the members of your cluster.
+This separation can be useful, for example, to use a virtual address for your REST API, with DNS round robin.
+
+To do so, you must specify different addresses for [`core.https_address`](server) (the address for the REST API) and [`cluster.https_address`](server) (the address for internal cluster traffic).
+
+1. Configure the bootstrap server:
+
+   `````{tabs}
+
+   ````{group-tab} Interactive configuration
+   When running `lxd init`, specify the address for the REST API when asked for the IP address or DNS name for the member (for example, `my.lxd.cluster:8443`).
+   This value is used for the `core.https_address` configuration.
+
+   After the initialization, set the `cluster.https_address` configuration to the address for internal cluster traffic.
+   For example:
+
+       lxc config set cluster.https_address 192.0.2.171:8443
+
+   ````
+   ````{group-tab} Preseed file
+   Specify the address for the REST API for the `core.https_address` configuration and the address for internal cluster traffic for the `cluster.https_address` configuration.
+
+   For example:
+
+   ```yaml
+   config:
+     core.https_address: my.lxd.cluster:8443
+     cluster.https_address: 192.0.2.171:8443
+     ...
+   ```
+
+   ````
+
+   `````
+
+1. Configure the joining members:
+
+   `````{tabs}
+
+   ````{group-tab} Interactive configuration
+   Before running `lxd init`, set the `core.https_address` configuration to the address for the REST API.
+   For example:
+
+       lxc config set core.https_address my.lxd.cluster:8443
+
+   Then run `lxd init` as usual.
+
+   ````
+   ````{group-tab} Preseed file
+   Specify the address for the REST API for the `core.https_address` configuration, in addition to the usual cluster configuration.
+   If you are using password authentication, specify the address for internal cluster traffic in the `cluster_address` field.
+
+   For example:
+
+   ```yaml
+   config:
+     core.https_address: my.lxd.cluster:8443
```

Comment:
Does that work? Or is everything but the cluster section ignored when joining a cluster?

Reply:
I'm not sure, if you use the same IP but a different port for the core.https_address you should be able to test this.